src/sedcli_main.c

```diff
@@ -293,6 +315,31 @@ int setpw_handle_opts(char *opt, char **arg)
 	return 0;
 }
 
+static int get_mbr_flag(char *mbr_flag)
+{
+	if (!strncasecmp(mbr_flag, "TRUE", 4))
+		return 1;
+
+	return 0;
+}
+
+bool mbr_enable = false;
+bool mbr_done = false;
+int mbr_control_handle_opts(char *opt, char **arg)
+{
+	if (!strcmp(opt, "device")) {
+		strncpy(opts->dev_path, arg[0], PATH_MAX - 1);
+	} else if (!strcmp(opt, "enable")) {
+		mbr_enable = true;
+		opts->enable = get_mbr_flag(arg[0]);
```

Comment:
getting segmentation fault when providing following input: ./sedcli -M -d /dev/nvme1n1 -e

Reply:
This is because a NULL character is being string compared...will fix it